charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/nodes.yaml

```diff
@@ -4,7 +4,7 @@ Do not change in-place! In order to change this file first read following link:
 https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack/hack
 */ -}}
 {{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
-{{- if and (or .Values.grafana.enabled .Values.grafana.forceDeployDashboards) (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) .Values.grafana.defaultDashboardsEnabled }}
+{{- if and (or .Values.grafana.enabled .Values.grafana.forceDeployDashboards) (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) .Values.grafana.defaultDashboardsEnabled .Values.nodeExporter.enabled }}
```

Comment:
https://github.com/prometheus-community/helm-charts/blob/b387aae0bb587bc6a97a35576d165b2d6bab8b0d/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/nodes.yaml#L3

Reply:
@monotek - sorry but I dont get your comment? Would u expand?

Reply:
@monotek that's what I did, see the change in charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py and rerun to generate this file